### DISCUSSION TITLE
cannot clone or push :Unsupported proxy syntax in 'socks:127.0.0.1:9150'

### DISCUSSION THREAD

**Discussion by utc425937** [community/community#52636] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am using github desktop and git for windows.
When cloning it shows :Cloning into 'C:\Users\...
fatal: unable to access 'https://github.com/.......git/': Unsupported proxy syntax in 'socks:127.0.0.1:9150'
9150 is tor listening port but i am not using tor that time neither do i use vpn or proxy,I did download tor and used it a couple weeks ago.
How can i solve that?

**Comment by opegamanalitarok** [comment]
It seems like Git is trying to use a SOCKS proxy that is no longer active. You can try disabling the proxy settings in Git to see if that resolves the issue.

Here are the steps to disable the proxy settings in Git:

1. Open Git Bash.
2. Type the following command to check the current Git proxy settings:

git config --global --get http.proxy

3. If the output shows a proxy, use the following command to unset the proxy:

git config --global --unset http.proxy

4. Similarly, check and unset the HTTPS proxy settings by running the following commands:

git config --global --get https.proxy
git config --global --unset https.proxy

5. Try cloning the repository again to see if the issue is resolved.

If this does not resolve the issue, you can also try reinstalling Git or GitHub Desktop to ensure that there are no lingering proxy settings.

Hope this helps! 🙌🙌

**Reply by gabrielcoce** [reply]
It works for me, 
Thanks!!!

**Reply by sksahilhaque** [reply]
Thanks!

**Reply by OlsenFischer** [reply]
Worked for me, Thanks a lot.

**Reply by vivekNinjatech** [reply]
It worked, Thanks!!!

**Reply by jaki2005** [reply]
I have no idea why it worked, but thanks a lot!!!

**Reply by Yanaa88** [reply]
Thank you!

**Comment by sahana-github** [comment]
@utc425937 try Updating proxy settings for Git
(git config --global http.proxy socks5h://127.0.0.1:9150)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @utc425937 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by om66id66** [comment]
git config --global --unset-all remote.origin.proxy

**Reply by webtrixztech** [reply]
working sukriya

**Comment by jkashmire** [comment]
The error message you're seeing usually pops up when there's a leftover proxy configuration that Git is trying to use. Even if you're not actively using Tor or a VPN now, sometimes these settings stick around.

Here's a quick fix to try: Open Git Bash and type git config --global --unset http.proxy and git config --global --unset https.proxy. This should clear out any proxy settings that might be causing trouble.

As for [rotating proxies](https://proxyrotator.com/rotating-proxies/) like those at Proxy Rotator, they can be handy for certain tasks that need frequent IP changes, like web scraping or testing. Just be sure to use them responsibly to avoid any unexpected hiccups like this one!

**Reply by MG72021** [reply]
[google.com](url)

**Comment by yusup-apandi** [comment]
thanks, it works🤩

**Comment by MuhamedHabib** [comment]
git config --global --unset-all remote.origin.proxy
but the easiest way to clone or push or any other command form anywhere is to use PAT access tokens if you've the right to access the needed repository by using for expamle::
git push https@YOUR PAT:://github/.../repo.git
